src/plugins/workspace/public/plugin.ts

```diff
@@ -593,6 +593,9 @@ export class WorkspacePlugin
 
     this.filterNavLinks(core);
 
+    // Unregister the page search that set by chrome service, since workspace has its own search
+    core.chrome.globalSearch.unregisterSearchCommand('pagesSearch');
```

Comment:
nit: better move it close to registerSearchCommand, it's more reable and have more conetxt of this unregister

Reply:
They are at two different function (`setup` vs `start`) so I guess we couldn't do so. 😆